File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let result=""
    for(item of s){
        if(item==" ")
            result+=item;
        else if(item==item.toUpperCase()){
            result+= String.fromCharCode((item.charCodeAt(0)-'A'.charCodeAt(0)+n)%26+'A'.charCodeAt(0));
        }
        else if(item==item.toLowerCase()){
            result+= String.fromCharCode((item.charCodeAt(0)-'a'.charCodeAt(0)+n)%26+'a'.charCodeAt(0));
        }
    }
    return result;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    calp="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    lalp=calp.lower()
    result=""
    for item in s:
        if item in calp:
            result+=calp[(calp.index(item)+n)%26]
        elif item in lalp:
            result+=lalp[(lalp.index(item)+n)%26]
        else:
            result+=item
    return result